tensorflow/core/ops/array_ops.cc

```diff
@@ -1491,6 +1491,7 @@ REGISTER_OP("Unique")
     .Output("idx: out_idx")
     .Attr("T: type")
     .Attr("out_idx: {int32, int64} = DT_INT32")
+    .Attr("auto_adjust_uniq_table_size: bool = false")
```

Comment:
I'm not pro exposing what should be an implementation detail as an attribute. I'd rather this be done under the covers without the user having to choose. If the user needs to choose, this could be a separate op too. But that would require some more exhaustive benchmarking.